packages/headless/src/utils/id.ts

```diff
@@ -3,7 +3,7 @@
  */
 // ⚠️ we make use of an IIFE to encapsulate the globalCounter so it can never accidentally be used somewhere else.
 const nextId = (() => {
-  let globalCounter = 0;
+  let globalCounter = 1;
```

Comment:
What's the difference? 😄

Reply:
Its less confusing if you take a look at the DOM in the dev tools, because with initial value 0 e.g. the active-descendant is 1 if you have focused the second option.

Functionally it does not make a difference